packages/lambdapi/lambdapi.2.6.0/opam

```diff
@@ -47,8 +47,6 @@ build: [
     "-j"
     jobs
     "@install"
-    "@runtest" {with-test}
-    "@doc" {with-doc}
```

Comment:
I believe this will prevent documentation from being generated for users who request it during installation. This could impact users who view documentation locally and I suspect it will prevent the generation of the documentation at https://ocaml.org/p/lambdapi/latest/doc/index.html  -- Is this intended?